TriLabelView/TriLabelView.swift

```diff
@@ -80,7 +80,11 @@ import UIKit
     }
     
     /// The font for the label text.
-    public var labelFont: UIFont = UIFont(name: "HelveticaNeue-Bold", size: 20)!
+    @IBInspectable public var labelFont: UIFont = UIFont.systemFont(ofSize: 15, weight: UIFontWeightBold) {
```

Comment:
You need to change this to UIFont.Weight.bold as `'UIFontWeightBold' has been renamed to 'UIFont.Weight.bold'`

Reply:
Ah, gotcha. Let me edit.